src/platform/apollolake/include/platform/lib/memory.h

```diff
@@ -292,7 +292,7 @@
 
 #define HEAP_BUFFER_SIZE	0x10000
 
-#define HEAP_BUFFER_BLOCK_SIZE	0x100
+#define HEAP_BUFFER_BLOCK_SIZE	0x80
```

Comment:
I would also place a static assert here for cache line size (all platforms) and HDA block size (CAVS platforms). This way we will throw up a build error if someone tries to change this in the future to an incompatible block size.